nlp4all/routes.py

```diff
@@ -587,7 +587,7 @@ def register_imc():
         hashed_password = bcrypt.generate_password_hash(
             fake_password).decode("utf-8")
         imc_org = Organization.query.filter_by(name="ATU").all()
-        a_project = imc_org[0].projects[0]
+        a_project = imc_org[0].projects[0] #error when no project. out of range TODO
```

Comment:
good catch

Reply:
I think that this was back when running Python 3.10